### PR TITLE
chore: Contact section added on README

### LINKED ISSUE #1377: Add Contact section on README
## Description

As a contributor,
I need to know where I can reach out to the community,
so that I can discuss or see discussions of this project.

I would suggest adding a section such as "Contact us" or "How to reach us" to the README.
An example is what is on [mentorship-backend README](https://github.com/anitab-org/mentorship-backend/).

### Update [Required]
- [ ] Add contact section with AnitaB.org Open Source Zulip link and #powerup stream link. 

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by Mehulipal**
### Description
Contributors need to know where they can reach out to the community, so that they can discuss or see discussions of this project. Hence, contact section has been added on README.

Fixes #1377 

### Type of Change:

- Documentation
- This change requires a documentation update (software upgrade on readme file)

### How Has This Been Tested?

Screenshot (180)

### Checklist:

&#9745;  My PR follows the style guidelines of this project
&#9745;  I have made corresponding changes to the documentation
&#9745;  My changes generate no new warnings 
&#9745;  New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Mehulipal** [comment]
Please review.

File: README.md [hunk 1]
```diff
@@ -65,3 +65,10 @@ Here's the link to the official documentation:
 1. Include spaces between parameters when you call a method for example: `Intent(MainActivity.this, GameActivity.class)`.
 1. Give relevant names to buttons and other resources. 
 1. Use `@id` instead of `@+id` when referring to resources that have been already created in xml files.
+
+## Contact Us
+
+You can reach our community and its maintainers on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/)
+
+If you are interested in contributing to the PowerUp application, we have a dedicated stream for this project where you can ask questions and interact with the community. Join with us on Zulip [here](https://anitab-org.zulipchat.com/#narrow/stream/222537-powerup)
```
Comment (isabelcosta) [inline README.md]:
```suggestion
If you are interested in contributing to the PowerUp application, we have a dedicated stream for this project where you can ask questions and interact with the community. Join with us on [#powerup](https://anitab-org.zulipchat.com/#narrow/stream/222537-powerup) stream on Zulip.
```

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@Mehulipal thank you so much for this contribution!
I left a small change request that I think could improve and make the message more clear :)
Once this is done, I am more than happy to approve.

**Comment by Mehulipal** [comment]
Ma'am, I have made the requested changes. Please review.

**Review by isabelcosta — APPROVED** [review]
Great work @Mehulipal 🎉

**Comment by isabelcosta** [comment]
@mayburgos can you approve/review this as well, since this is a small change to the readme?

**Comment by Mehulipal** [comment]
@mayburgos  kindly review the changes

**Review by sunjunkie — APPROVED** [review]

**Comment by atm1504** [comment]
@Mehulipal the branch name you used has no connection with the issue you resolved, please prefer to use a proper branch name. You should try to follow such guidelines to work and maintain a large project.
Moreover, the title of the pr should start with```chore``` not ```feat```.

**Comment by Mehulipal** [comment]
@atm1504 I have made the requested changes in PR name. I will surely keep your advice in mind and will name my branch accordingly from next time. Thanks for your review :)

**Comment by isabelcosta** [comment]
great work @Mehulipal 🎉

**Pull request merged**